### PR TITLE
readme: add help for submodule method of working

### PR DESCRIPTION
**Pull request by illwieckz**
add help for submodule method of working

### CODE REVIEW

**Comment by illwieckz** [comment]
It may be easier to read to review the change this way:

- https://github.com/Unvanquished/Unvanquished/blob/illwieckz/submodule-help/README.md#dealing-with-submodules

**Comment by illwieckz** [comment]
I made this a wiki page: https://wiki.unvanquished.net/wiki/Dealing_with_submodules

I'll simplify the text in README and add a link to the wiki.

**Comment by slipher** [comment]
Does this really belong in the README at all? It's some random Git usage tips not specific to our project. Except for the `/sync` thing, but that only applies for a handful of project members.

**Comment by illwieckz** [comment]
I simplified the wording for the README, almost everything else is now on wiki instead.

**Comment by slipher** [comment]
LGTM

File: README.md [hunk 1]
```diff
@@ -264,3 +264,11 @@ You then start a game server with your custom mod this way:
 ```
 
 See [above instructions](#configuring-the-server) about configuring the server and the wiki page for [running a server](https://wiki.unvanquished.net/wiki/Server/Running) providing more details, including ways to provide fast HTTP download for your mod.
+
+## Dealing with submodules
+
+Please don't commit submodules in pull requests, as the submodule references may not exist after the pull requests are rebased and merged.
+
+When a feature is split across more than one repository, submodules are expected to be committed after each pull requests have been merged.
```
Comment (slipher) [inline README.md:272]: `s/have been/has been/`

Comment (illwieckz) [reply README.md:272]: done

**Pull request merged**